packages/core/src/application.ts

```diff
@@ -59,6 +59,8 @@ export class Application extends Context {
 
     this.handleHttp = (req: ServerRequest, res: ServerResponse) =>
       this._handleHttpRequest(req, res);
+
+    this.bind('logError').to(this._logError.bind(this));
```

Comment:
Should we start to externalize the keys to a separate ts file?

Should we name it as something like `loggers:error-logger`?